spec/cc/engine/analyzers/javascript/main_spec.rb

```diff
@@ -124,6 +124,14 @@
   def engine_conf
     CC::Engine::Analyzers::EngineConfig.new({
       'config' => {
+        'checks' => {
+          'similar-code' => {
+            'enabled' => true,
+          },
+          'identical-code' => {
+            'enabled' => true,
```

Comment:
I hope we're going to exercise both the default (no `config.checks` key) as well as a config that overrides that default?